internal/pkg/describe/app.go

```diff
@@ -23,13 +23,14 @@ import (
 
 // App contains serialized parameters for an application.
 type App struct {
-	Name      string                   `json:"name"`
-	Version   string                   `json:"version"`
-	URI       string                   `json:"uri"`
-	Envs      []*config.Environment    `json:"environments"`
-	Services  []*config.Workload       `json:"services"`
-	Jobs      []*config.Workload       `json:"jobs"`
-	Pipelines []*codepipeline.Pipeline `json:"pipelines"`
+	Name         string                   `json:"name"`
+	Version      string                   `json:"version"`
+	URI          string                   `json:"uri"`
+	Envs         []*config.Environment    `json:"environments"`
+	Services     []*config.Workload       `json:"services"`
+	Jobs         []*config.Workload       `json:"jobs"`
+	Pipelines    []*codepipeline.Pipeline `json:"pipelines"`
+	WorkloadEnvs map[string][]string      `json:"workloadenvs"`
```

Comment:
Do we want these information in JSON output?

My take is that we probably want to just show them in HumanString for now, and not JSON.

Reply:
Made sure that Marshal ignores this field for JSON output using `json"-"`